compiler/noirc_evaluator/src/ssa/ir/function.rs

```diff
@@ -85,6 +86,8 @@ impl Function {
     /// Note that any parameters or attributes of the function must be manually added later.
     pub(crate) fn new(name: String, id: FunctionId) -> Self {
         let mut dfg = DataFlowGraph::default();
+        // Adds root node for the location tree
+        dfg.add_location_to_root(Location::dummy());
```

Comment:
Is this something that could be done in `DataFlowGraph::default()` itself?

Reply:
Done